lesson11/js/main.js

```diff
@@ -0,0 +1,70 @@
+"use strict";
+const menuContainer = document.getElementById("js-menu_container");
+const menuLists = document.createElement("ul");
+const loading = document.getElementById("js-loading");
+
+function showLoadingImg() {
+  const loadingImg = document.createElement("img");
+  loadingImg.src = "images/loading-circle.gif";
+  loadingImg.classList.add("el_loadingImg");
+
+  loading.appendChild(loadingImg);
+}
+
+function createMenuList(values) {
+  const fragment = document.createDocumentFragment();
+
+  Object.keys(values).forEach((key) => {
+    values[key].forEach(value => {
```

Comment:
Please check this code.
I'm worried about the foreach array processing code, so please confirm.

Reply:
[IMO](https://codesandbox.io/s/hidden-river-39jjb?file=/js/main.js)
I used the destructuring assignment (分割代入)
You can look into it if you want.😄

Reply:
[IMO](https://codesandbox.io/s/dazzling-architecture-yk1o3?file=/js/main.js)
I tried using Destructuring assignment(分割代入)
You can look into it if you want😄

It took me a long time to check the operation because of the bad chrome.
I'm sorry I've kept you waiting.😥

Reply:
Thank you for review.

I know Destructuring assignment for the first time.
I tried using Destructuring assignment and was able to write the code simply.
Using forEach twice made it harder to read which values ​​were passed, so the code was easier to read.

I will try more array operations using Destructuring assignment.

[d8c39d9](https://github.com/nyattsu72/hanson-frontend-cource/pull/14/commits/d8c39d9ffd35b34182f09dd846778c8f08dedb8c)